docs/projects.md

```diff
@@ -14,7 +14,13 @@ Create a new Project
 ```
 USAGE
   $ dvc projects create [--config-path <value>] [--auth-path <value>] [--repo-config-path <value>] [--client-id
-    <value>] [--client-secret <value>] [--project <value>] [--no-api] [--headless]
+    <value>] [--client-secret <value>] [--project <value>] [--no-api] [--headless] [--key <value>] [--name <value>]
+    [--description <value>]
+
+FLAGS
+  --description=<value>  Description for display in the dashboard.
+  --key=<value>          Unique ID to refer to the environment.
+  --name=<value>         Human readable name of the environment.
```

Comment:
could these be confusing flag names if they're just `key` and `name` but refer to the environment key and names instead of project key and names? it makes sense for the environments commands but not here? I think it was added in the `createCommand.ts` file but could it just be in environments instead?

Reply:
yeah need to change the description of each flag, but they refer to the actual object's name and key